velox/expression/fuzzer/ExpressionFuzzer.h

```diff
@@ -299,23 +299,25 @@ class ExpressionFuzzer {
   // Returns random integer between min and max inclusive.
   int32_t rand32(int32_t min, int32_t max);
 
-  static const inline std::string kTypeParameterName = "T";
-
   const Options options_;
 
   std::vector<CallableSignature> signatures_;
   std::vector<SignatureTemplate> signatureTemplates_;
 
+  // IMPORTANT: this needs to be sanitized because ExpressionFuzzer sanitizes
+  // type names before adding them to the maps below.
+  static const inline std::string kTypeParameterName = exec::sanitizeName("T");
+
   /// Maps the base name of a return type signature to the function names that
-  /// support that return type. Base name could be "T" if the return type is a
-  /// type variable.
+  /// support that return type. Base name could be `kTypeParameterName` if the
+  /// return type is a type variable.
   std::unordered_map<std::string, std::vector<std::string>>
       typeToExpressionList_;
 
   /// Maps the base name of a *concrete* return type signature to the function
   /// names that support that return type. Those names then each further map
   /// to a list of CallableSignature objects that they support. Base name
-  /// could be "T" if the return type is a type variable.
+  /// could be `kTypeParameterName` if the return type is a type variable.
```

Comment:
nit: Could you help check whether expressionToSignature_ can contain kTypeParameterName as a key? From the code, it seems like those keys are added to expressionToTemplatedSignature_ instead. If so, let's correct this comment.